dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/SignIn/BaseEmailConfirmationPageModel.cs

```diff
@@ -0,0 +1,80 @@
+using System.ComponentModel.DataAnnotations;
```

Comment:
Not sure if this is the best place for this to live

Reply:
Makes sense to me 👍